src/main/java/gift/controller/LoginPageController.java

```diff
@@ -0,0 +1,29 @@
+package gift.controller;
+
+import gift.config.KakaoOauthConfig;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/login")
+public class LoginPageController {
+    private final KakaoOauthConfig kakaoOauthConfig;
+
+    public LoginPageController(KakaoOauthConfig kakaoOauthConfig) {
+        this.kakaoOauthConfig = kakaoOauthConfig;
+    }
+
+    @GetMapping("/page")
+    public String loginPage(Model model) {
+        String url = String.format(
+                "https://kauth.kakao.com/oauth/authorize?response_type=code&client_id=%s&redirect_uri=%s",
+                kakaoOauthConfig.getClientId(),
+                kakaoOauthConfig.getRedirectURI()
+        );
```

Comment:
```suggestion
        URI uri = UriComponentsBuilder.fromUriString("https://kauth.kakao.com")
                .path("/oauth/authorize")
                .queryParam("response_type", "code")
                .queryParam("client_id", apiKey)
                .queryParam("redirect_uri", redirectUrl)
                .build().toUri();
```
URI를 만들 때는 builder를 활용하면 구조적으로 만들 수 있어 더 명확하고 안전하게 생성이 가능합니다!